ArduPlane/Parameters.cpp

```diff
@@ -1303,6 +1303,14 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("RNGFND_LND_ORNT", 36, ParametersG2, rangefinder_land_orient, ROTATION_PITCH_270),
 #endif
+
+    // @Param: GUID_TIMEOUT
```

Comment:
All the existing guided parameters use the prefix GUIDED_,

Reply:
That came from Copter. Looks like Plane and Copter chose different prefixes for the same mode.

Reply:
I matched `GUIDED_` prefix. For some reason the other GUIDED_ parameters are part of plane and not ModeGuided.